Make it more clear you need to kill chasers (#7628)
endless-sky/endless-sky: data/remnant/remnant 2 cognizance.txt

mission "Remnant: Cognizance 25"
	name "Patrol systems near Postverta."

> 	description "Gavriil has asked you to do a patrol through the nearby systems. Along the way, make sure that a Palavret that passed by recently did not leave behind any surprises."

Reviewer: ```suggestion
	description "Gavriil has asked you to do a patrol through the nearby systems to clear out any Korath presence."
```

Author: I see what you mean, I think it can be changed differently though.

> 		dialog "You have destroyed the ships that the Palavret left behind."

Reviewer: ```suggestion
		dialog "You have destroyed the ships that the Palavret left behind. Time to finish your patrol of the region."
```

Author: You could finish the patrol without killing the chasers, however the purpose of this PR is to make sure that doesn’t happen by making it a little clearer.

	on waypoint

> 		dialog "You have patrolled all six systems."

Reviewer: ```suggestion
		dialog "You have patrolled all six systems. Time to return to <planet>."
```

Author: There are two requirements to the mission. If you just patrol with the out killing the chasers, it won’t mark as complete. That was the entire point of this pill request.

	on enter Egaria

> 		dialog "As soon as you enter the system, three 'olofezes begin targeting your ship. Better take care of them...

Reviewer 2: ```suggestion
		dialog "As soon as you enter the system, three 'olofezes begin targeting your ship. Better take care of them..."
```
You forgot the closing quote

> 		dialog "You have landed on <planet> but have not destroyed the ships the Palavret left behind. Better depart and go find them."

Reviewer 3: This assumes that you know that those ships are there.
I suspect this message would also show if you return to Ssil Vida before visiting all 6 systems.
if explicitly pointing the ships is desired, I would suggest instead using an `on enter` for the system they are in pointing them out to the player (I suspect it is possible for the player to jump in and out without those ships targeting them if they are far enough away). That could also be used to set a condition that could perhaps be used in the `on visit` to decide whether or not the player should reasonably know that there are ships to destroy, or instead just hasn't visited every system yet.

Author: yeah that makes sense to me

> 		dialog "As soon as you enter the system, three 'olofezes begin targeting your ship. These are probably the sort of surprises that Gavriil was worried about."

Reviewer: ```suggestion
		dialog "As soon as you enter the system, three 'olofezes begin targeting your ship. These are probably the surprises Gavriil was worried about."
```

Author: I think “sort of” can be removed but “that” is fine.
